timescaledb/getting-started/create-hypertable.md

```diff
@@ -1,90 +1,81 @@
+is PostgreSQL, all the regular PostgreSQL tables, indexes, etc. can be created right 
+alongside your TimescaleDB hypertables. This makes creating and working with TimescaleDB 
+tables fairly similar to how you would with just vanilla PostgreSQL. 
+
+## Hypertables and chunks
+Hypertables and chunks make storing and querying times-series data fast at petabyte 
+scale. 
+
+TimescaleDB automatically partitions time-series data into **chunks**, or sub-tables, 
+based on time and space. You can configure chunk size so that recent chunks fit in memory 
+for faster queries. 
+
+A **hypertable** is an abstraction layer over chunks that hold the time-series data.
+Hypertables allow you to query and access data from all the chunks within the hypertable.
+You get all the benefits of automatic chunking for time-series data, alongside the simplicity
+of working with what looks like a standard, single PostgreSQL table.
 
 Hypertables and chunks enable superior performance for shallow and wide queries,
 like those used in real-time monitoring. They are also good for deep and narrow
 queries, like those used in time-series analysis.
 
-<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/illustration-hypertable-chunk.png" alt="hypertable and chunks"/>
+<img class="main-content__illustration" 
+src="https://s3.amazonaws.com/assets.timescale.com/docs/images/getting-started/hypertables-chunks.png" 
+alt="hypertable and chunks"/>
+
+<highlight type="note">
+For more detailed information, check out the [Overview page on 'Hypertables and chunks'](/timescaledb/latest/overview/core-concepts/hypertables-and-chunks/).
+</highlight>
 
-You can interact with chunks individually if you need to, but chunks are created
-automatically based on the `chunk_time` and `chunk_size` parameters you specify.
 
 **Create your first hypertable**
 
 ```sql
-----------------------------------------
--- Hypertable to store weather metrics
-----------------------------------------
--- Step 1: Define regular table
-CREATE TABLE IF NOT EXISTS weather_metrics (
-
-   time TIMESTAMP WITHOUT TIME ZONE NOT NULL,
-   timezone_shift int NULL,
-   city_name text NULL,
-   temp_c double PRECISION NULL,
-   feels_like_c double PRECISION NULL,
-   temp_min_c double PRECISION NULL,
-   temp_max_c double PRECISION NULL,
-   pressure_hpa double PRECISION NULL,
-   humidity_percent double PRECISION NULL,
-   wind_speed_ms double PRECISION NULL,
-   wind_deg int NULL,
-   rain_1h_mm double PRECISION NULL,
-   rain_3h_mm double PRECISION NULL,
-   snow_1h_mm double PRECISION NULL,
-   snow_3h_mm double PRECISION NULL,
-   clouds_percent int NULL,
-   weather_type_id int NULL
+CREATE TABLE stocks_real_time (
+   time TIMESTAMPTZ NOT NULL,
+   symbol TEXT NOT NULL,
+   price DOUBLE PRECISION NULL,
+   day_volume INT NULL
 );
 
--- Step 2: Turn into hypertable
-SELECT create_hypertable('weather_metrics','time');
+SELECT create_hypertable('stocks_real_time','time');
+```
+
```

Comment:
```suggestion

When you create a hypertable partitioned on time, TimescaleDB will automatically create an index on the time column for you. However, you'll often filter your time-series data on other columns as well and using indexes appropriately will help your queries perform well.

Since we will often be querying the stock trade data by the company symbol, we should add an index for it. We'll include the time column because time-series data often looks for data in a specific period of time.

```sql
CREATE INDEX ix_symbol_time ON stocks_real_time (symbol, time DESC);
```